LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt

```diff
@@ -4,14 +4,14 @@ layer at (0,0) size 800x98
   RenderBlock {HTML} at (0,0) size 800x98
     RenderBody {BODY} at (8,8) size 784x82
       RenderAttachment {ATTACHMENT} at (1,1) size 266x80
-        RenderFlexibleBox {DIV} at (0,0) size 266x80 [bgcolor=#0000000D]
+        RenderFlexibleBox {DIV} at (0,0) size 266x80 [bgcolor=#0000000C]
```

Comment:
Actually, I believe this should stay `#D`, as expected by online tests.
I also get `#C` when running tests locally, I'm not sure why... I should really investigate! 😅

Reply:
@squelart the system color definitions probably changed between OS versions.